api/src/wfl/module/wgs.clj

```diff
@@ -47,20 +47,19 @@
     {:cram_ref_fasta       cram_ref_fasta
      :cram_ref_fasta_index (str cram_ref_fasta ".fai")}))
 
-(defn make-references
+(def ^:private default-references
   "HG38 reference, calling interval, and contamination files."
-  [prefix]
   (let [hg38 "gs://gcp-public-data--broad-references/hg38/v0/"]
     (merge references/contamination-sites
-      (references/hg38-genome-references prefix)
+      (references/hg38-genome-references nil)
```

Comment:
Could we `(def hg38-genome-references …)` and just merge it as data?

Reply:
> Not sure I understand the motivation, but looks good to me.

The idea is that I think it's weird to specify all references via a reference_fasta_prefix - surely that should only define the reference fasta?